src/collective/volto/formsupport/restapi/services/form_data/csv.py

```diff
@@ -83,8 +88,12 @@ def get_data(self):
                 # add fixed columns values
                 value = item.attrs.get(k, None)
                 data[k] = json_compatible(value)
+            if "waiting_list" in custom_colums:
+                data.update({"waiting_list": not (index < limit)})
+
```

Comment:
ok, quindi decidi di gestirla con waiting => true /false.
va bene. non abbiamo indicazioni in merito. però è per non informatici. metterei 'Sì' / 'No'

Reply:
Fatto